sdks/java/extensions/ml/build.gradle

```diff
@@ -44,11 +44,30 @@ dependencies {
     testRuntimeOnly project(":runners:google-cloud-dataflow-java")
 }
 
-project.test {
-    def gcpProject = project.findProperty("gcpProject") ?: 'apache-beam-testing'
-    include "**/**IT.class"
-    def pipelineOptions = [
-            "--project=${gcpProject}"
-    ]
-    systemProperty "beamTestPipelineOptions", JsonOutput.toJson(pipelineOptions)
+/**
+ * These are integration tests with the GCP ML services and the DirectRunner.
+ */
+task integrationTest(type: Test) {
+  group = "Verification"
+  def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
+  def gcpTempRoot = project.findProperty('gcpTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests'
+  systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
+          "--runner=DirectRunner",
+          "--project=${gcpProject}",
+          "--tempRoot=${gcpTempRoot}",
+  ])
+
+  // Disable Gradle cache: these ITs interact with live service that should always be considered "out of date"
+  outputs.upToDateWhen { false }
```

Comment:
Just want to note that "UP-TO-DATE" and "FROM-CACHE" are two different Gradle statuses for a task. I presume if it is never up to date it is also never cached...

Reply:
I copied this from https://github.com/apache/beam/blob/master/sdks/java/extensions/google-cloud-platform-core/build.gradle#L76-L77

I can update the comments in both places to be more correct - it sounds like just dropping "Disable Gradle cache:" would address it?